package.json

```diff
@@ -19,6 +19,7 @@
   "license": "ISC",
   "dependencies": {
     "camelcase": "1.0.2",
+    "co": "^4.6.0",
```

Comment:
I agree; initially I updated and fixed all the deps, but I'd rather do that as a totally separate PR, so I just went with the flow here. Want to make sure I can get :login and :push in without breaking any existing docker users, so minimal changes.